native/membrane_videocompositor_wgpu/src/lib.rs

```diff
@@ -1,13 +1,34 @@
+use rustler::ResourceArc;
+
+mod compositor;
+
 #[derive(Debug, rustler::NifStruct)]
-#[module = "Membrane.VideoCompositor.OpenGL.Implementations.Native.Rust.RawVideo"]
-struct RawVideo {
-    width: usize,
-    height: usize,
-    pixel_format: rustler::Atom,
+#[module = "Membrane.VideoCompositor.Implementations.OpenGL.Native.Rust.RawVideo"]
+pub struct RawVideo {
```

Comment:
Can't we just use `u32` and convert to usize when we need that? I think it's unnatural to keep those parameters in `usize`.